modules/cli/src/main/scala/scala/cli/commands/fix/Fix.scala

```diff
@@ -0,0 +1,117 @@
+package scala.cli.commands.fix
+
+import caseapp.core.RemainingArgs
+
+import java.nio.charset.StandardCharsets
+
+import scala.build.input.ElementsUtils.projectSettingsFiles
+import scala.build.input.Inputs
+import scala.build.internal.{Constants, CustomCodeWrapper}
+import scala.build.preprocessing.PreprocessedSource._
+import scala.build.preprocessing.{DirectivesPositions, PreprocessedSource, ScopePath}
+import scala.build.{CrossSources, Logger, Position, Sources}
+import scala.cli.commands.shared.SharedOptions
+import scala.cli.commands.{ScalaCommand, SpecificationLevel}
+
+object Fix extends ScalaCommand[FixOptions] {
+  override def group                   = "Main"
+  override def scalaSpecificationLevel = SpecificationLevel.RESTRICTED
```

Comment:
Might be safer for a while, we can move it to `RESTRICTED` after it proves to be stable.
```suggestion
  override def scalaSpecificationLevel = SpecificationLevel.EXPERIMENTAL
```